cro-clib/src/jsonrpc.rs

```diff
@@ -194,6 +194,7 @@ unsafe fn create_rpc(
     progress_callback_user: CroProgressPtr,
     user_data: *const std::ffi::c_void,
 ) -> Result<CroJsonRpc> {
+    let _ = env_logger::try_init();
```

Comment:
`try_init` returns a `Result`. Can we handle the error or just use `init`?

Reply:
Because `create_rpc` can be called multiple times in one process, so we have to use `try_init` and ignore the result (it returns error when called multiple times).

Reply:
I think it'll be "Err" if it was already initialised, so it should be ignored (maybe a comment can be added to explain that).
For `init`, it'll panic if it's called multiple times if I'm not mistaken

Reply:
> I think it'll be "Err" if it was already initialised, so it should be ignored (maybe a comment can be added to explain that).
> For `init`, it'll panic if it's called multiple times if I'm not mistaken

yes, added comments now

Reply:
how about separating as another api?

Reply:
there's no `env_logger::deinit()` though.
after `env_logger::init()`, use still can disable log by environment variables.